python/lsst/ip/isr/isr.py

```diff
@@ -345,6 +345,8 @@ def overscanCorrection(ampMaskedImage, overscanImage, fitType='MEDIAN', order=1,
         diff = numpy.abs(biasArray - medianBiasArr[:, numpy.newaxis])
         biasMaskedArr = numpy.ma.masked_where(diff > collapseRej*stdevBiasArr[:, numpy.newaxis], biasArray)
         collapsed = numpy.mean(biasMaskedArr, axis=1)
+        if collapsed.mask.sum() > 0:
+            collapsed.data[collapsed.mask] = numpy.mean(biasArray.data[collapsed.mask], axis=1)
```

Comment:
I realize you are adding the original values back into the masked pixels in order to plot them below, but might this be a gotcha for anyone later using `collapsed.data` and expecting the non-modified values?  I'm not familiar with the object here, so am not sure what the masked pixels would've been assigned in `collapsed.data`, so forgive my naïveté if this is a non-issue!